website/static/js/notificationsTreebeard.js

```diff
@@ -165,16 +141,21 @@ function ProjectNotifications(data) {
                                 onchange: function(ev) {
                                     subscribe(item, ev.target.value);
                                 }},
-                                [
-                                    m('option', {value: 'none', selected : item.data.event.notificationType === 'none' ? 'selected': ''}, 'Never'),
-                                    m('option', {value: 'email_transactional', selected : item.data.event.notificationType === 'email_transactional' ? 'selected': ''}, 'Instantly'),
-                                    m('option', {value: 'email_digest', selected : item.data.event.notificationType === 'email_digest' ? 'selected': ''}, 'Daily')
-                            ])
+                                options)
                         ]);
                     }
                 });
             }
             else {
+                options = [
+                    m('option', {value: 'none', selected : item.data.event.notificationType === 'none' ? 'selected': ''}, 'Never'),
+                    m('option', {value: 'email_transactional', selected : item.data.event.notificationType === 'email_transactional' ? 'selected': ''}, 'Instantly'),
+                ];
+                if (item.data.event.title !== 'mailing_list_events') {
+                    options.push(m('option', {value: 'email_digest', selected : item.data.event.notificationType === 'email_digest' ? 'selected': ''}, 'Daily'));
+                    options.push(m('option', {value: 'adopt_parent', selected: item.data.event.notificationType === 'adopt_parent' ? 'selected' : ''},
+                                                 'Adopt setting from parent project ' + displayParentNotificationType(item)));
+                }
```

Comment:
Fixed in https://github.com/CenterForOpenScience/osf.io/pull/5645/commits/8729daf5d5844f694cb3ae1db068644bc87cd3e1